### DISCUSSION TITLE
Acceleration Tower - what to do first, acceleration or jerk?

### DISCUSSION THREAD

**Discussion by unitrix0** [5axes/Calibration-Shapes#140] [Q&A]
What should I do first with the acceleration tower, jerk or acceleration? And what do I look for when I run a jerk test?

**Comment by 5axes** [comment]
You can try Jerk then Acceleration.  You must check the visual result on the part , you can see some ringing wave or also issue on the corner. But this test is not as simple to interpret as a temperature tower.

![image](https://github.com/5axes/Calibration-Shapes/assets/11015345/dbbba246-2dc3-44f0-8f4d-d69f9bf45808)
